src/targets/github.ts

```diff
@@ -122,13 +127,23 @@ export class GitHubTarget extends BaseTarget {
       };
     }
 
+    const { data: latestRelease } = await this.github.repos.getLatestRelease({
```

Comment:
Stupid question. What happens here?:

- actual latest release is v3.0
- previous release published was v1.1
- you're now publishing v2.2

Reply:
2.2 will _not_ be marked as latest. It fetches the release that is currently marked as "latest" on Github (3.0), and if that has a version > than the version we are trying to publish, it does not mark it as latest!